### DISCUSSION TITLE
Automate the change of admin password

### DISCUSSION THREAD

**Discussion by sendrakratos** [goharbor/harbor#19346] [Q&A] [answered]
Hello,

I would like to know if it's possible to automate the change of admin password.

I have seend that only it's possible to change admin password on web interface:
https://goharbor.io/docs/main/administration/managing-users/reset-user-password

But i need to change the password with API or other script. I can't use web interface if I want to automate this change.

Is there some option??

Thanks you, best regards.

**Comment by chlins** [comment, marked as answer]
The web interface do it by calling the harbor API, you can access the user API to change the password. FYI: https://github.com/goharbor/harbor/blob/26a4f6eeea7b6387f3d6a162d99475a078792704/api/v2.0/swagger.yaml#L5684
